client-core/src/transaction_builder/default_transaction_builder.rs

```diff
@@ -104,7 +104,7 @@ where
         let mut witnesses = Vec::with_capacity(transaction.inputs.len());
 
         for input in &transaction.inputs {
-            let input = wallet_client.output(&input.id, input.index)?;
+            let input = wallet_client.output(&input.id, input.index as usize)?;
```

Comment:
Change the `output()` function to take `u64` instead?

Reply:
currently, RUST iterator index is also usize,
also other output functions are in usize,
so it seems ok as usize